lib/chef/provider/chef_group.rb

```diff
@@ -15,13 +15,13 @@ def whyrun_supported?
       if differences.size > 0
         description = [ "update group #{new_resource.name} at #{rest.url}" ] + differences
         converge_by description do
-          rest.put("groups/#{new_resource.name}", normalize_for_put(new_json))
+          rest.put("organizations/#{new_resource.org}/groups/#{new_resource.name}", normalize_for_put(new_json))
```

Comment:
@jkeiser Thanks for the clarification on the Chef pattern. It makes sense and I will back off this PR after we figure the API for adding members in organization in one-shot.